advanced/list-of-log-names.md

```diff
@@ -73,4 +73,4 @@ This category is for debugging purposes. Logs in this category are meant to aid
 
 | Name                | Emitted when     |
 | ------------------- | ---------------- |
-| REACTION_ROLE_DEBUG | Someone uses a reaction role |
+| REACTION_ROLE_DEBUG | A user triggers a reaction role, and something goes wtong |
```

Comment:
Too vague and there's a typo. This log is triggered, say, when permissions are misconfigured or someone tries to get a join-only role or similar things